src/connectRoutes.js

```diff
@@ -115,8 +115,8 @@ export default (
   }
 
   const {
-    location: locationKey = 'location',
-    title: titleKey = 'title',
+    selectLocationState = state => state.location,
```

Comment:
oh my bad, not in addition. just the original 2 like u did. everything is looking perfect, including the docs.

...as long as u do `if typeof === 'function'` like u did flow will be able to infer.

...i saw the comment about the `kind`. that must be a new flow issue. each update to flow, it gets better and can find more things. Do you mind just checking the existence of the `kind` key rather than using `$FlowIssue`.